src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,25 @@
+// import { useEffect } from "react";
```

Comment:
If this is commented out, you likely don't need it yet. Please don't upload code before it's needed.